### DISCUSSION TITLE
［灌水］怎么回事？法厄同已经从绳网消失整整74天了！

### DISCUSSION THREAD

**Discussion by bushigemengsishuihaiyinga** [share121/inter-knot#4843] [灌水]
鉴于绳网用户都听过传奇绳匠「法厄同」的名号，我就直接开门见山吧。他已经从绳网上消失整整一周了！「法厄同」虽然不会每天都接取委托，但一周至少也得有个三四次，正是在持之以恒的努力下才成为了业界传奇。但现在是怎么回事？大佬不会厌倦了绳匠生活、准备隐退江湖了吧？

**Comment by Oumainory** [comment]
听说之前防卫军有委托过法厄同，不会是被抓了吧<img src="https://bbs-static.miyoushe.com/static/2024/03/21/36606bdafa3ea2537746b44f38bc5856_6388510732715759806.png" alt="丽都漫步-11号" width="75" >

**Comment by huameidudu** [comment]
**退隐江湖？**_不至于吧..._

**Comment by K-k423** [comment]
据说法厄同抢了个U盘，被失主定位了IP，治安局不知道从哪里得到的消息，把人给抓了
